parse/special.go

```diff
@@ -152,9 +153,21 @@ Switch:
 				p.helpOverview()
 				break
 			}
-			p.helpAbout(tok.Text)
+			text := tok.Text
+			if tok.Type == scan.String {
+				if s, err := strconv.Unquote(text); err == nil {
+					text = s
+				}
+			}
+			p.helpAbout(text)
```

Comment:
I suggest that instead of duplicating things here, put it in a function in helpDocs and call it at the top of help and helpAbout.